dali/pipeline/operators/decoder/nvjpeg/decoupled_api/nvjpeg_decoder_decoupled_api_test.cc

```diff
@@ -54,7 +54,7 @@ class nvjpegDecodeDecoupledAPITest : public GenericDecoderTest<ImgType> {
   }
 
  private:
-  unsigned int hybrid_huffman_threshold_;
+  unsigned int hybrid_huffman_threshold_ = -1;
```

Comment:
Isn't it weird to initialize unsigned int to -1?

Reply:
Done